src/main/cli/settings.c

```diff
@@ -1624,6 +1625,10 @@ const clivalue_t valueTable[] = {
     { "box_user_3_name", VAR_UINT8 | MASTER_VALUE | MODE_STRING, .config.string = { 1, MAX_BOX_USER_NAME_LENGTH, STRING_FLAGS_NONE }, PG_MODE_ACTIVATION_CONFIG, offsetof(modeActivationConfig_t, box_user_3_name) },
     { "box_user_4_name", VAR_UINT8 | MASTER_VALUE | MODE_STRING, .config.string = { 1, MAX_BOX_USER_NAME_LENGTH, STRING_FLAGS_NONE }, PG_MODE_ACTIVATION_CONFIG, offsetof(modeActivationConfig_t, box_user_4_name) },
 #endif
+
+#if defined(USE_RX_MSP_OVERRIDE)
+    { "msp_override_passthrough_channels", VAR_UINT32 | MASTER_VALUE, .config.minmaxUnsigned = {0, UINT32_MAX}, PG_RX_MSP_OVERRIDE_CONFIG, offsetof(rxMspOverrideConfig_t, passthrough_mask)},
```

Comment:
Is this meant to allow an override of up to 32 channels?

Reply:
That was the idea, yes. Would some sort of array be more appropriate for the configuration?